### DISCUSSION TITLE
What is the use of the `index.txt` files for each page generated?

### DISCUSSION THREAD

**Discussion by taheramr** [vercel/next.js#57222] [App Router]
I'm using the app router with the `export` mode for generating an SSG website, I'm trying to debug why the `index.txt` file that gets fetched in the network is HUGE, its around 10 megabytes, I wanna understand how this file is generated and what its main usage in the nextjs building phase?

**Comment by icyJoseph** [comment]
Is that the file that has the RSC strings? Maybe you need to re-architect how you pass props, because you might be duplicating data into the RSC.

**Reply by davetapley** [reply]
Yep, see here: https://github.com/vercel/next.js/discussions/59394#discussioncomment-8844056
